plugins/woocommerce-beta-tester/src/remote-inbox-notifications/data/actions.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import { apiFetch } from '@wordpress/data-controls';
+import { controls } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import TYPES from './action-types';
+import { API_NAMESPACE, STORE_KEY } from './constants';
+
+/**
+ * Initialize the state
+ *
+ * @param {Array} notifications
+ */
+export function setNotifications( notifications ) {
+	return {
+		type: TYPES.SET_NOTIFICATIONS,
+		notifications,
+	};
+}
+
+export function setLoadingState( isLoading ) {
+	return {
+		type: TYPES.SET_IS_LOADING,
+		isLoading,
+	};
+}
+
+export function setNotice( notice ) {
+	return {
+		type: TYPES.SET_NOTICE,
+		notice,
+	};
+}
+
+export function* importNotifications( notifications ) {
+	try {
+		yield apiFetch( {
+			method: 'POST',
+			headers: {
+				'Content-Type': 'application/json',
+			},
+			path: `${ API_NAMESPACE }/remote-inbox-notifications/import`,
+			body: JSON.stringify( notifications ),
+		} );
+
+		yield controls.dispatch(
+			STORE_KEY,
+			'invalidateResolutionForStoreSelector',
+			'getNotifications'
+		);
+	} catch ( error ) {
+		throw new Error();
+	}
+}
+
+export function* deleteNotification( id ) {
+	try {
+		yield apiFetch( {
+			method: 'DELETE',
+			path: `${ API_NAMESPACE }/remote-inbox-notifications/${ id }`,
+		} );
+
+		yield {
+			type: TYPES.DELETE_NOTIFICATION,
+			id,
+		};
+	} catch {
+		throw new Error();
```

Comment:
Is it intended to not pass the original error?

Reply:
Not intended 😄 Copied and pasted from a different code base. I'll make an update.

Reply:
Fixed it in https://github.com/woocommerce/woocommerce/pull/48735/commits/d47969fb8113c6b40c6e509ebbc66bc8d4aea00f